blue_bringup/config/controllers.yaml

```diff
@@ -0,0 +1,125 @@
+### Controller for publishing to the joint_states topic
+
+joint_state_controller:
+  type: joint_state_controller/JointStateController
+  publish_rate: 125
+
+### Gripper controller
+
+gripper_controller:
+  type: effort_controllers/GripperActionController
+  joints:
+    - $(arg side)_gripper_joint
+  joint: $(arg side)_gripper_joint
+  goal_tolerance: 0.01
+  stalled_velocity_threshold: 0.01
+  stall_timeout: 0.2
+  gains:
+    $(arg side)_gripper_joint: {p: 15, i: 5, d: 10.5, i_clamp_min: -1, i_clamp_max: 1}
+
+### Joint controllers
+
+joint_trajectory_controller:
+  type: effort_controllers/JointTrajectoryController
+  joints:
+    - $(arg side)_base_roll_joint
+    - $(arg side)_shoulder_lift_joint
+    - $(arg side)_shoulder_roll_joint
+    - $(arg side)_elbow_lift_joint
+    - $(arg side)_elbow_roll_joint
+    - $(arg side)_wrist_lift_joint
+    - $(arg side)_wrist_roll_joint
+  constraints:
+    goal_time: 0.5
+    $(arg side)_base_roll_joint:
+      trajectory: 0.5
+      goal: 0.02
+    $(arg side)_shoulder_lift_joint:
+      trajectory: 0.5
+      goal: 0.02
+    $(arg side)_shoulder_roll_joint:
+      trajectory: 0.5
+      goal: 0.02
+    $(arg side)_elbow_lift_joint:
+      trajectory: 0.5
+      goal: 0.02
+    $(arg side)_elbow_roll_joint:
+      trajectory: 0.5
+      goal: 0.02
+    $(arg side)_wrist_lift_joint:
+      trajectory: 0.5
+      goal: 0.02
+    $(arg side)_wrist_roll_joint:
+      trajectory: 0.5
+      goal: 0.02
+  gains:
+    $(arg side)_base_roll_joint: {p: 750, i: 150, d: 5, i_clamp_min: -500, i_clamp_max: 500}
```

Comment:
 Can we decrease the aggression on these gains?

Reply:
yeah, i'll just drop them arbitrarily and we can tune later?

Reply:
sounds good